lib/auth/grpcserver.go

```diff
@@ -5436,6 +5440,22 @@ func NewGRPCServer(cfg GRPCServerConfig) (*GRPCServer, error) {
 		stableUNIXUsersServiceServer,
 	)
 
+	scopedAccessControl, err := accesscontrol.New(accesscontrol.Config{
+		Authorizer: cfg.Authorizer,
+	})
+	if err != nil {
+		return nil, trace.Wrap(err, "creating scoped access control service")
+	}
+	scopedrolev1.RegisterScopedRoleServiceServer(server, scopedAccessControl)
+
+	scopedProvisioning, err := provisioning.New(provisioning.Config{
+		Authorizer: cfg.Authorizer,
+	})
+	if err != nil {
+		return nil, trace.Wrap(err, "creating scoped provisioning service")
+	}
+	scopedtokenv1.RegisterScopedTokenServiceServer(server, scopedProvisioning)
```

Comment:
Which name is canonical: "scoped provisioning service" or "scoped token service"? We should stick to one.

Reply:
Sure, would it make sense to use "scoped token service" for the variable name then or is there a good reason not to?